pandas/tests/frame/test_analytics.py

```diff
@@ -2073,3 +2073,14 @@ def test_n_duplicate_index(self, df_duplicates, n, order):
         result = df.nlargest(n, order)
         expected = df.sort_values(order, ascending=False).head(n)
         tm.assert_frame_equal(result, expected)
+
+    def test_series_broadcasting(self):
+        df = DataFrame({'A': [np.nan, 2.0, np.nan]})
```

Comment:
can you add the issue number as a comment here. (and a 1-line comment that this is a smoke test for numpy warnings)